setup.py

```diff
@@ -2,59 +2,57 @@
 # -*- coding: utf-8 -*-
 
 """The setup script."""
-from glob import glob
-import os
-from os.path import basename
-from os.path import splitext
-
 from setuptools import setup, find_packages
-from sphinx.setup_command import BuildDoc
+
+try:
+    from sphinx.setup_command import BuildDoc
+
+    cmdclass = {"build_sphinx": BuildDoc}
+except ImportError:
+    # sphinx not installed - do not provide build_sphinx cmd
+    cmdclass = {}
+
 
 with open("README.rst") as readme_file:
     readme = readme_file.read()
 
 with open("HISTORY.rst") as history_file:
     history = history_file.read()
 
+REQUIREMENTS = [
+    "libecl",
+    "numpy",
+    "pandas>0.23.0",
+    "pyyaml>=5.1",
+    "six>=1.12.0",
+]
 
-def relpath(*args):
-    """Return path of args relative to this file"""
-    root = os.path.dirname(__file__)
-    if isinstance(args, str):
-        return os.path.join(root, args)
-    return os.path.join(root, *args)
-
-
-def parse_requirements(filename):
-    """Load requirements from a pip requirements file"""
-    try:
-        lineiter = (line.strip() for line in open(filename))
-        return [line for line in lineiter if line and not line.startswith("#")]
-    except IOError:
-        return []
-
-
-REQUIREMENTS = parse_requirements("requirements.txt")
-
-SETUP_REQUIREMENTS = ["pytest-runner", "setuptools>=28", "setuptools_scm"]
+SETUP_REQUIREMENTS = ["setuptools>=28", "setuptools_scm"]
 
-TEST_REQUIREMENTS = parse_requirements("requirements_dev.txt")
+TEST_REQUIREMENTS = [
+    "flake8>=2.6.0",
+    "pyarrow",
```

Comment:
I think pyarrow must be deleted here, to have travis test both with and without pyarrow.

Reply:
Will do (also note that the `dev` requirement list is already a bit shorter).